src/java/org/apache/cassandra/utils/streamhist/StreamingTombstoneHistogramBuilder.java

```diff
@@ -88,15 +91,15 @@ public StreamingTombstoneHistogramBuilder(int maxBinSize, int maxSpoolSize, int
      *
      * @param point the point to be added
      */
-    public void update(int point)
+    public void update(long point)
```

Comment:
The class JavaDoc should be updated to reflect the changes

Reply:
Done

Reply:
Great, thanks. There is another outdated reference to `SSTableReader#getDroppableTombstonesBefore(int)` on the class JavaDoc

Reply:
Done